python/ray/rllib/agents/agent.py

```diff
@@ -605,7 +620,9 @@ def session_creator():
             callbacks=config["callbacks"],
             input_creator=input_creator,
             input_evaluation_method=config["input_evaluation"],
-            output_creator=output_creator)
+            output_creator=output_creator,
+            remote_worker_envs=remote_worker_envs,
+            entangled_worker_envs=entangled_worker_envs)
```

Comment:
Should use config (and remove entangled_)

Reply:
Commented about config above, and entangled does not exist (comment is on outdated snippet).